### ISSUE TITLE
Soft target in classification?

### ISSUE THREAD

**Issue by guileryu01** [dmlc/mxnet#3332]
Does mxnet support "soft target" in classification tasks? i.e for each sample, instead of a label specifying a single class, I have a target distribution over all classes. The network is then trained to minimize some sort of loss function (e.g. cross entropy loss) between model prediction and the target?

**Comment by piiswrong** [comment]
softmaxoutput supports this. just pass in n dimensional labels

**Comment by guileryu01** [comment]
Thanks, managed to make it work after modifying Accuracy metric.
